Yoast/wordpress-seo
# Update snippet preview image

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

In Gutenberg and classic editor:

* Create a new post.
* Mobile snippet preview should have no image.
* Add a featured image.
* Mobile snippet preview should have featured image.
* Remove featured image.
* Mobile snippet preview should have no image.
* Add image to content.
* Mobile snippet preview should have content image.
* Add another image to content.
* Mobile snippet preview should have the first content image.
* Add a featured image.
* Mobile snippet preview should have featured image.

## UI changes

## Documentation

## Quality assurance


Fixes #13661
